src/util.ts

```diff
@@ -789,4 +789,15 @@ export function killTree(processId: number): void {
 		} catch (err) {
 		}
 	}
-}
\ No newline at end of file
+}
+
+export function getRootWorkspaceFolder(forPathUri: vscode.Uri): vscode.WorkspaceFolder {
+	let workspaceFolder = vscode.workspace.getWorkspaceFolder(forPathUri);
+	let previousWorkspaceFolder: vscode.WorkspaceFolder;
+	while (workspaceFolder !== undefined) {
```

Comment:
Can you explain the scenario that would need this loop?
Taking your example of aws, if `github.com/aws/aws-sdk-go` is already part of the workspace and we run the command for `github.com/aws/aws-sdk-go/service/rds/rdsutils`, wouldnt `vscode.workspace.getWorkspaceFolder(forPathUri)` give you the uri of the parent workspace folder? And isnt that enough?